cupy/core/raw.pyx

```diff
@@ -432,10 +499,11 @@ cdef class RawModule:
 
 
 @cupy.util.memoize(for_each_device=True)
-def _get_raw_module(str code, str path, tuple options=(), str backend='nvrtc',
-                    bint translate_cucomplex=False,
-                    bint enable_cooperative_groups=False,
-                    tuple name_expressions=None, log_stream=None):
+def _get_raw_module(str code, str path, tuple options, str backend,
+                    bint translate_cucomplex,
+                    bint enable_cooperative_groups,
+                    tuple name_expressions,
+                    object log_stream):
```

Comment:
Note: I didn't realize until now that `log_stream` becomes part of the cache key, which is a little odd since it doesn't affect the generated kernels. However, I don't see any real harm here, as in most use cases I expect `log_stream=None`.